runtime/doc/filetype.txt

```diff
@@ -547,6 +549,10 @@ q                         :quit if invoked as $MANPAGER, otherwise :close.
 Variables:
 *g:no_man_maps*             Do not create mappings in manpage buffers.
 *g:ft_man_folding_enable*   Fold manpages with foldmethod=indent foldnestmax=1.
+*b:man_default_sections*    A comma seperated list of default sections to priortize
+                          for the current buffer.
```

Comment:
typo; also I recommend omitting articles (a, the) unless it greatly increases clarity. They're usually just noise in these types of annotated lists.

    Comma-separated list of preferred sections.

"current buffer" never needs mention for a `b:` variable.